examples/cpp/asymmetric/asymmetric-sync.cpp

```diff
@@ -0,0 +1,80 @@
+#include <catch2/catch_test_macros.hpp>
+#include <cpprealm/sdk.hpp>
+
+static const std::string APP_ID = "cpp-tester-uliix";
+
+namespace realm {
+// :snippet-start: asymmetric-object
+struct WeatherSensorReading {
+  realm::primary_key<realm::object_id> _id{realm::object_id::generate()};
+  std::string deviceId;
+  double temperatureInFahrenheit;
+  int64_t windSpeedInMph;
+};
+REALM_ASYMMETRIC_SCHEMA(WeatherSensorReading, _id, deviceId,
+                        temperatureInFahrenheit, windSpeedInMph)
+// :snippet-end:
+}  // namespace realm
+
+TEST_CASE("Asymmetric object example", "[write][sync]") {
+  // :snippet-start: connect-and-authenticate
+  auto appConfig = realm::App::configuration();
+  appConfig.app_id = APP_ID;
+  auto app = realm::App(appConfig);
+  auto user = app.login(realm::App::credentials::anonymous()).get();
+  // :snippet-end:
+  // :snippet-start: open-asymmetric-synced-realm
+  auto syncConfig = user.flexible_sync_configuration();
+  auto realm = realm::open<realm::WeatherSensorReading>(syncConfig);
+  // :snippet-end:
+
+  // Doing this for simplicity in testing, but should not be relevant to
+  // dev code examples, so hiding it.
+  const realm::object_id oid = realm::object_id::generate();
+
+  // :snippet-start: create-asymmetric-object
+  auto weatherSensorReading =
+      realm::WeatherSensorReading{._id = oid,  // :remove:
```

Comment:
looks like you need to update the indentation on the other properties too, they're _really_ indented
```suggestion
      realm::WeatherSensorReading{
      ._id = oid,  // :remove:
```